Android--EN/Displayer.java

```diff
@@ -371,7 +371,7 @@ private Bitmap transformBase64toBitmap(String base64String) {
      *
      * @since 2.11.0
      *
-     * This method disables the user from moving or zooming the view through touch screen gestures.
+     * This method disables the user from moving or zooming the view through touch-screen gestures. 
```

Comment:
The user can adjust the view mainly through touch-screen gestures because the API is for devices running Android, such as mobile phones and tablets. For these devices, usually mouses and keyboards are not needed.